From b8c554aeaf05e656c07b60cceea47ec60c4a8e6e Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sat, 10 Jun 2017 03:24:42 -0500
Subject: [PATCH] Use member initializer list for nSpace2
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Compilation of object file proteus/mprans/RANS3PF.o fails with the following
error on an old compiler:

In file included from proteus/mprans/RANS3PF.cpp:452:
proteus/mprans/RANS3PF2D.h:410: error: ISO C++ forbids initialization of member ‘nSpace2’
proteus/mprans/RANS3PF2D.h:410: error: making ‘nSpace2’ static error: command
'gcc' failed with exit status 1

This patch fixes the issue by moving the initialization of variable nSpace2 to
the constructor using initializer list.
---
 proteus/mprans/RANS3PF2D.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/proteus/mprans/RANS3PF2D.h b/proteus/mprans/RANS3PF2D.h
index 0fe21e1ce..700786558 100644
--- a/proteus/mprans/RANS3PF2D.h
+++ b/proteus/mprans/RANS3PF2D.h
@@ -407,9 +407,10 @@ namespace proteus
   public:
     cppHsuSedStress<2> closure;
     const int nDOF_test_X_trial_element,
-      nSpace2=4;
+      nSpace2;
     CompKernelType ck;
     cppRANS3PF2D():
+      nSpace2(4),
       closure(150.0,
               0.0,
               0.0102,
